language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/definitions/FunctionDefinition.java

```diff
@@ -30,21 +30,21 @@
         return new Object[][] {
                 {"emptyLinePrimitiveDataTypes.json", "function"},
                 {"nonEmptyLinePrimitiveDataTypes.json", "function"},
-                {"userDefinedStructEmptyLine.json", "function"},
-                {"userDefinedStructNonEmptyLine.json", "function"},
+                {"userDefinedRecordEmptyLine.json", "function"},
+                {"userDefinedRecordNonEmptyLine.json", "function"},
                 {"userDefinedFunctionsEmptyLine.json", "function"},
                 {"userDefinedFunctionsNonEmptyLine.json", "function"},
                 {"importPackagesEmptyLine.json", "function"},
                 {"importPackagesNonEmptyLine.json", "function"},
                 {"allVisibleSymbolsEmptyLine.json", "function"},
                 {"languageConstructsEmptyLine.json", "function"},
                 {"languageConstructsNonEmptyLine.json", "function"},
-                {"enumSuggestAssignment1.json", "function"},
-                {"enumSuggestAssignment2.json", "function"},
-                {"enumSuggestVarDef1.json", "function"},
-                {"enumSuggestVarDef2.json", "function"},
-                {"structFields.json", "function"},
-                {"structBoundFunctionsAndFields.json", "function"},
+//                {"enumSuggestAssignment1.json", "function"},
+//                {"enumSuggestAssignment2.json", "function"},
+//                {"enumSuggestVarDef1.json", "function"},
+//                {"enumSuggestVarDef2.json", "function"},
+//                {"structFields.json", "function"},
+//                {"structBoundFunctionsAndFields.json", "function"},
```

Comment:
can we remove these as they are no longer valid?

Reply:
Will keep until the latest syntax related tests added